janitor/config/test.py

```diff
@@ -1,27 +1,26 @@
-# flake8: noqa
-from .defaults import *
+from .defaults import *  # noqa: F401, F403
 
 # setting here will overwrite those in 'defaults.py'
 SYNC_JOB_INTERVAL_SEC = 300
 SYNC_JOB_OVERLAP_SEC = 10
 
-LABWHERE_DB: DbConnectionDetails = {  # type: ignore
-    "db_name": "janitor_tests_lw_input",
-    "host": "127.0.0.1",
-    "port": 3306,
-    "username": "root",
-    "password": "",
-}
+LABWHERE_DB = DbConnectionDetails(  # noqa: F405
```

Comment:
You could avoid this `noqa` if you add the `from janitor.types import DbConnectionDetails` line to the top of this file.